### ISSUE TITLE
About the training question

### ISSUE THREAD

**Issue by AMzhanghan** [zhuyu-cs/MeanFlow#23]
I'm using the finetune_fm branch. I first trained without CFG, and the loss decreased very slowly around the 30th epoch. Afterwards, I loaded this model for finetuning, but the finetuning loss was the same, decreasing very slowly. What is the reason? All training is in fp32.

**Comment by zhuyu-cs** [comment]
Please make sure your hyperparameters for cfg finetuning are set correctly. You'll need to reference the cfg training hyperparameters from meanflow-xl on the main branch. Also, the loss is just a reference. You can test the FID after finetuning for 50k or 100k steps to verify the performance after finetuning.
